marketplace/commercetools-products/src/index.js

```diff
@@ -0,0 +1,37 @@
+import 'whatwg-fetch';
+import 'regenerator-runtime/runtime';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import '@contentful/forma-36-react-components/dist/styles.css';
+import '@contentful/forma-36-fcss/dist/styles.css';
+import { init, locations } from 'contentful-ui-extensions-sdk';
+
+import { CommerceToolsField } from './field.js';
```

Comment:
i'd also optionally argue in favor of removing the `.js` extension from all import statements

Reply:
Done. But if this is the style we're going for, we should enforce it in `@contentful/exlint-config-extension` using https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md